doc/user/layouts/shortcodes/supportLevel.html

```diff
@@ -1,4 +1,6 @@
-<svg class="level-of-support level-{{ .Get 0 }}" xmlns="http://www.w3.org/2000/svg">
-  <use href="#icon-{{ .Get 0 }}" />
-</svg>
-{{ replace (.Get 0) "-" " " | title }}
+<div style="width: 100%; height: 100%; text-align: center;">
+  <svg class="level-of-support level-{{ .Get 0 }}" xmlns="http://www.w3.org/2000/svg">
+    <use href="#icon-{{ .Get 0 }}" />
+  </svg>
+  <p>{{ replace (.Get 0) "-" " " | title }}</p>
```

Comment:
when you wrap the text in a <p> tag the font size goes from 14px (applied to <td>) to 15px (applied to <p>) intentional?